pupil_src/shared_modules/calibration_routines/gaze_mappers.py

```diff
@@ -94,6 +116,7 @@ def map_batch(self, pupil_list):
 
     def on_pupil_datum(self, p):
         self._caches[p["id"]].append(p)
+        temporal_cutoff = 2 * self.calculate_temporal_cutoff(*self._caches)
```

Comment:
@romanroibu 
`calculate_temporal_cutoff` stores the result in `self.temporal_cutoff`, but in the end we are using 2* the result.
Why do we have this difference between `self.temporal_cutoff` and the actual cutoff being used in the end?

Reply:
I agree, the variable and function should be renamed to `self.recently_estimated_framerate` and `estimate_framerate()` respectively, and then `temporal_cutoff = 2 * estimate_framerate(...)`

Reply:
That makes a lot more sense.

Reply:
On it